src/commands/convertConnections/convertRemoteConnections.ts

```diff
@@ -0,0 +1,63 @@
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.md in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { type AppSettingTreeItem } from "@microsoft/vscode-azext-azureappsettings";
+import { RoleAssignmentExecuteStep, UserAssignedIdentityListStep } from "@microsoft/vscode-azext-azureutils";
+import { AzureWizard, type AzureWizardExecuteStep, type AzureWizardPromptStep, type IActionContext } from "@microsoft/vscode-azext-utils";
+import { type MessageItem } from "vscode";
+import { localize } from "../../localize";
+import { type SlotTreeItem } from "../../tree/SlotTreeItem";
+import { createActivityContext } from "../../utils/activityUtils";
+import { pickFunctionApp } from "../../utils/pickFunctionApp";
+import { ConvertSettingsStep } from "./ConvertSettingsStep";
+import { type IConvertConnectionsContext } from "./IConvertConnectionsContext";
+import { SelectConnectionsStep, type Connection } from "./SelectConnectionsStep";
+
+export async function convertRemoteConnections(context: IActionContext, node?: AppSettingTreeItem): Promise<void> {
+    const connections: Connection[] = []
+    if (node) {
+        await node.runWithTemporaryDescription(context, localize('converting', 'Converting...'), async () => {
+            await convertRemoteConnectionsInternal(context, connections, node);
+        });
+    } else {
+        await convertRemoteConnectionsInternal(context);
+    }
+}
+
+export async function convertRemoteConnectionsInternal(context: IActionContext, connections?: Connection[], node?: AppSettingTreeItem): Promise<void> {
```

Comment:
Are these internal functions identical besides setting local to false?

Reply:
There are some slight differences beyond that but once I split up the ConvertSettingsStep into remote and local they will also use different execute steps.